spec/bonita/resources/bdm/business_data_resource_spec.rb

```diff
@@ -25,7 +25,7 @@
 
   describe "#search" do
     let(:path) do
-      "/bonita/API/bdm/businessData/com.company.model.Employee?c=10&f=firstName%3DJohn&f=lastname%3DDoe&p=0&q=findEmployeeByFirstNameAndLastName&p=0" # rubocop:disable Metrics/LineLength
```

Comment:
now faraday just looks for the path and it fails otherwise.
There is a strict mode to check params but bonita params contain several times the same key which cannot be written as a ruby hash